multicloud/src/main/java/de/uniulm/omi/cloudiator/sword/multicloud/pricing/aws/AWSProductAndTerms.java

```diff
@@ -0,0 +1,10 @@
+package de.uniulm.omi.cloudiator.sword.multicloud.pricing.aws;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class AWSProductAndTerms {
+    public AWSProduct product;
```

Comment:
public?

Reply:
I'm gonna leave it as is - it's just a pojo